image/tools/lib/component/codeready_pv.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+function pv_dump_data {
```

Comment:
can you move it to lib/utils.sh and source it in the entrypoint.sh file? this way other components can use this function (just from an organization perspective)

Reply:
Will do

Reply:
note, it has `codeready` namespace hard-coded, don't forget to parametrize it if you move to utils :)

Reply:
and Path in the container is hard coded as well ;)